src/HW/LedgerConnect/LedgerConnect.tsx

```diff
@@ -274,7 +273,7 @@ class _LedgerConnect extends React.Component<Props, State> {
               contentContainerStyle={styles.flatListContentContainer}
               data={devices}
               renderItem={({item}: {item: Device}) => (
-                <DeviceItem device={item} onSelect={() => this._onSelectDevice(item)} />
+                <DeviceItem disabled={waiting} device={item} onSelect={() => this._onSelectDevice(item)} />
```

Comment:
Disable the option to click on a `DeviceItem` when we're connecting to Ledger via Bluetooth to avoid race conditions and connecting to a new device while we're connecting to a different device.

Reply:
this prevents double clicking, or selecting multiple devices